pkg/corerp/renderers/container/render.go

```diff
@@ -109,6 +109,20 @@ func (r Renderer) GetDependencyIDs(ctx context.Context, dm v1.DataModelInterface
 		}
 	}
 
+	for _, envvars := range properties.Container.Env {
```

Comment:
Nit: I'd suggest a comment here describing why we need to do all of this. Something like: `Environment variables might reference a `secretStores` resource, so walk through them and find any references`.